### PR TITLE
Sensor tests speedup

### PR DESCRIPTION
**Pull request by gast04**
speed up sensor test case, from ~23s to ~11s

### CODE REVIEW

File: selfdrive/sensord/tests/test_sensord.py [hunk 1]
```diff
@@ -116,10 +116,14 @@ def setUpClass(cls):
...
-  @with_processes(['sensord'])
+    # read initial sensor values every test case can use
+    managed_processes["sensord"].start()
+    cls.events = read_sensor_events(3)
+    managed_processes["sensord"].stop()
+
   def test_sensors_present(self):
     # verify correct sensors configuration
-    events = read_sensor_events(10)
+    events = self.events
```
Comment (adeebshihadeh) [inline selfdrive/sensord/tests/test_sensord.py:126]: replace all the local events with self.events

File: selfdrive/sensord/tests/test_sensord.py [hunk 2]
```diff
@@ -116,10 +116,14 @@ def setUpClass(cls):
     # make sure gpiochip0 is readable
     HARDWARE.initialize_hardware()
 
-  @with_processes(['sensord'])
+    # read initial sensor values every test case can use
+    managed_processes["sensord"].start()
+    cls.events = read_sensor_events(3)
```
Comment (adeebshihadeh) [inline selfdrive/sensord/tests/test_sensord.py:121]:
```suggestion
    cls.events = read_sensor_events(5)
```

**Review by adeebshihadeh — COMMENTED** [review]
Why not use for `test_sensor_values_sanity_check`?

File: selfdrive/sensord/tests/test_sensord.py (hunk 2 above)
Comment (gast04) [reply selfdrive/sensord/tests/test_sensord.py:121]: gonna need to do a quick test run with that change, check if it doesnt screw with the values

**Comment by gast04** [comment]
> 

cause if we increase it the miniamal sample check wouldnt make sense anymore, as we would receive more than
twice as much, so I skipped that one for now

**Pull request merged**